source/org/zfin/profile/presentation/ChromosomalLocationBean.java

```diff
@@ -0,0 +1,92 @@
+package org.zfin.profile.presentation;
+
+import org.zfin.mapping.GenomeLocation;
+
+public class ChromosomalLocationBean {
+    Long ID;
+    String entityID;
+    String assembly;
+    String chromosome;
+    Integer startLocation;
+    Integer endLocation;
+
+    public static ChromosomalLocationBean fromGenomeLocation(GenomeLocation persistedLocation) {
+        ChromosomalLocationBean clBean = new ChromosomalLocationBean();
+        clBean.setID(persistedLocation.getID());
+        clBean.setEntityID(persistedLocation.getEntityID());
+        clBean.setAssembly(persistedLocation.getAssembly());
+        clBean.setChromosome(persistedLocation.getChromosome());
+        clBean.setStartLocation(persistedLocation.getStart());
+        clBean.setEndLocation(persistedLocation.getEnd());
+        return clBean;
+    }
+
+    public Long getID() {
```

Comment:
use
@Setter
@Getter
instead of writing them by hand. Keeps the class smaller.

Reply:
Done.